Kickstarter-iOS/Views/Controllers/AddNewCardViewController.swift

```diff
@@ -58,6 +62,12 @@ STPPaymentCardTextFieldDelegate, MessageBannerViewControllerPresenting {
     let navigationBarButton = UIBarButtonItem(customView: self.saveButtonView)
     self.navigationItem.setRightBarButton(navigationBarButton, animated: false)
 
+    self.zipcodeView.textField.addTarget(self, action: #selector(zipcodeTextFieldDoneEditing),
```

Comment:
I was just thinking that would be a better practice to avoid accessing IBOutlets from outside the class, and treat `SettingsFormFieldView` as a TextField representation, instead of an UIView container. Since the zipcodeTextField represents a textField, this approach would let the class implement the `UITextFieldDelegate` messages itself, keeping it in its own context. The tradeoff is the implementation of the `SettingsFormFieldViewDelegate`, in other hand, though, we could get rid of the `addTarget` methods. But this was just a suggestion and a different idea to solve the same problem.

Reply:
I hear you, and if this view is going to be reused in a bunch of places it would be nice to get rid of the `addTarget` methods. I'd say if we end up using this view again we can re-think where the`UITextFieldDelegate` methods should live.